doc/api/next_api_changes/deprecations/26917-AL.rst

```diff
@@ -0,0 +1,5 @@
+``ContourLabeler.add_label_clabeltext``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+... is deprecated.  `.ContourLabeler.add_label` now sets
+`.Text.set_transform_rotates_text` based on the value of *use_clabeltext*
+passed to the `.ContourLabeler` constructor.
```

Comment:
The latter part of this note belongs in a behaviour change note, I think.

Reply:
Sure.